supbot/texts.py

```diff
@@ -0,0 +1,4 @@
+START_MESSAGE = """
+Привет!
+Я xi.supbot!
+"""
```

Comment:
suggestion: использовать словарики, будет удобнее к примеру добавить команды в менюшке бота, если ввести `/`
```suggestion
COMMANDS = {
    '/help': 'узнать возможности бота',
    '/cat': 'фотка котика',
}
"""
```

добавить команды:
```py
from aiogram import Bot
from aiogram.types import BotCommand

from lexicon.lexicon import COMMANDS


async def set_main_menu(bot: Bot):
    main_menu_commands = [
        BotCommand(
            command=command,
            description=description
        ) for command, description in COMMANDS.items()
    ]
    await bot.set_my_commands(main_menu_commands)
```

результат:
![Снимок](https://github.com/xi-effect/xi.auth/assets/145784290/e1e3ca04-16b8-4699-a6a8-5cda5bdbb767)

Reply:
ну и в целом удобнее crudить текста / команды

Reply:
С описанием команд в боте приветсвенное сообщение никак не связано (оно будет строк 10). Но в целом описание команд добавить можно. В твоём примере функция `set_main_menu` где вызывается?

Reply:
ну я просто как на будущее, приветсвенное сообщение тоже можно добавить в словарик как
 ```py
 '/start' : START_MESSAGE
 ```
 а set_main_menu вызывается при инициализации, к примеру тут maybe_initialize_telegram_app